static/js/components/dashboard/CoursePrice.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+
+import type { Course } from '../../flow/programTypes';
+import {
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_OFFERED_NOT_ENROLLED,
+} from '../../constants';
+import { formatPrice } from '../../util/util';
+
+export default class CoursePrice extends React.Component {
+  props: {
+    course: Course,
+  };
+
+  render() {
+    const { course } = this.props;
+    let firstRun = {};
+    let price = null;
+    let enrollmentStatus = null;
+
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
+        if (firstRun.price) {
+          price = <span className="course-price-display">{formatPrice(firstRun.price)}</span>;
+        }
+        enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
```

Comment:
Enrollment is not always open, it depends on the `enrollment_start_date`. We want to display it only if the date is equal or later than the current day (right @pdpinch?). See `CourseDescription` for an example of handling dates. Also, if the user is already enrolled it doesn't make sense to show `Enrollment open`.